05/DB.test.js

```diff
@@ -1 +1,165 @@
-import DB from './DB';
\ No newline at end of file
+import DB from './DB';
+
+describe('insert', () => {
+    it('should inform if ID is not a number', () => {
+        expect.assertions(1);
+        const db = new DB();
+        const data = {
+            id: '3'
+        }
+        const promise = db.insert(data);
+        return promise.catch(err => {
+            expect(err).toBe('ID can be only number!')
+        })
+    })
+
+    it('should inform if ID is not a number', () => {
+        expect.assertions(1);
+        const db = new DB();
+        db._rows = [{
+            name: "name",
+            id: 3
+        }]
+        const data = {
+            id: 3
+        }
+        const promise = db.insert(data);
+        return promise.catch(err => {
+            expect(err).toBe('ID can\'t be duplicated!')
+        })
+    })
+
+    it('should return ID when it is a number', () => {
+        expect.assertions(1);
+        const db = new DB();
+        const data = {
+            id: 3
+        }
+        const promise = db.insert(data);
+        return promise.then(result => {
+            expect(result).toBe(data)
+        })
+    })
+
+})
+
+describe('select', () => {
+    it('should return row when give an ID', () => {
+        expect.assertions(1);
+        const db = new DB();
+        db._rows = [{ // czy powinnam przekazac id poprzez db.insert???
+            name: "name",
+            id: 3
+        }, {
+            name: "name",
+            id: 5
+        }]
+        const id = 5;
+        const promise = db.select(id)
+        return promise.then(result => {
+            expect(result).toBe(db._rows[1]) //?? Nie miałam innego pomyslu :(
+        })
+    })
+    it('should inform when ID is not found', () => {
+        expect.assertions(1);
+        const db = new DB();
+        db._rows = [{
+            name: "name",
+            id: 3
+        }]
+        const id = 1;
+        const promise = db.select(id);
+        return promise.catch(err => {
+            expect(err).toBe('ID not found')
+        })
+    })
+})
+describe('remove', () => {
+    it('should return when ID is found', async () => {
+        const db = new DB();
+        db._rows = [{
+            name: "name",
+            id: 3
+        }, {
+            name: "name",
+            id: 5
+        }]
+        const id = 5;
+        const result = await db.remove(id)
+        expect(result).toBe('Item was remove!')
```

Comment:
Można też sprawdzić czy teraz w bazie jest tylko 1 element przez pobranie ich za pomocą `.getRows()`

Reply:
Zrobiłam to w jednym teście i zastanawiam się czy można tak łączyć

Reply:
Tak można łączyć, ale pamiętaj, aby testować jeden przypadek np. usuwanie. 
Tutaj sprawdzasz to przez 2 warunki czy jest odpowiedni komunikat oraz czy pozostała odpowiednia liczba danych.